xim/convdisp.cpp

```diff
@@ -140,17 +134,21 @@ create_default_fontset(const char *im_lang, const char *locale) {
 	setlocale(LC_CTYPE, locale);
 
     if (!strcmp(im_lang, "ja"))
-	name = fontset_ja;
+	name = uim_scm_symbol_value_str("uim-xim-fontset-ja");
     else if (!strcmp(im_lang, "ko"))
-	name = fontset_ko;
+	name = uim_scm_symbol_value_str("uim-xim-fontset-ko");
     else if (!strcmp(im_lang, "zh_CN"))
-	name = fontset_zhCN;
+	name = uim_scm_symbol_value_str("uim-xim-fontset-zh-cn");
     else if (!strcmp(im_lang, "zh_TW:zh_HK"))
-	name = fontset_zhTW;
+	name = uim_scm_symbol_value_str("uim-xim-fontset-zh-tw-and-zh-hk");
```

Comment:
I think that `zh_TW:zh_HK` means that `zh_TW` or `zh_HK`.
How about renaming to `...-zh-tw-or-zh-hk`?

Reply:
I thought that `zh_TW:zh_HK` was Chinese (Traditional) langgroup to which `zh_TW` and `zh_HK` match. So, the fontset definition would apply to both `zh_TW` and `zh_HK`. Would renaming it to just `...zh-tw` be alright? `zh_TW` is the primary lang code of the `zh_TW:zh_HK` langgroup.
https://github.com/uim/uim/blob/b9262f7/scm/i18n.scm#L198-L243

Reply:
Renaming to `...-zh-tw` is OK to me.
Could you rename to it? Then I'll merge this.

Reply:
Thank you! By the way, do my force-pushes reset the code review process for you?

Reply:
No problem.
I'll merge this.